code/Data exploration.R

```diff
@@ -258,8 +322,43 @@ data <- data %>%
   left_join(solar_nsw,by=c('DATE'))
 
 
-##Deduplicate
-data <- data %>% distinct()
+#######################
+##Join public holiday##
+#######################
+
+nsw_public_holiday <- holiday_aus(year(min(data$DATE)):year(max(data$DATE)), state = "NSW") %>%
+  rename(DATE=date)
+
+data <- data %>%
+  left_join(nsw_public_holiday,by=c('DATE'))
```

Comment:
I'm getting a many to many join here - can u pls review?
![image](https://github.com/llong-unsw/project-group-L/assets/143413975/fdde4864-92f4-49a2-b6e1-20494812919c)

Reply:
Fixed, base package was weird..